comicboxd/app/schema/series-query.go

```diff
@@ -102,7 +102,7 @@ func (r SeriesResolver) Read() int32 {
 func (r SeriesResolver) Tags() []string {
 	data := r.s.TagsJSON
 	tags := []string{}
-	if data == nil {
+	if len(data) == 0 {
```

Comment:
Does replacing the existing check vs. adding a new one have any implications?

This is a part of the source I am unfamiliar with, can `data` still be `nil`, if so won't getting rid of that check make it blow up?

Reply:
Cool